### PR TITLE
Only show companies associated with current user's team (rough setup)

### PR DESCRIPTION
**Pull request by KNWR**
Trello ticket 121 - see commented questions on PipelineContainer.tsx

### CODE REVIEW

File: client/containers/PipelineContainer.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { ChildContainer } from '../lib/combineContainers';
 
 export interface PipelineContainerState {
   currentPartner: string;
+  currentTeam: string; // 1/ can I use enum Team here, or stick w strings?
```
Comment (KNWR) [inline client/containers/PipelineContainer.tsx]: Can I use enum Team here, or stick w string type? i.e. can I import Team

File: client/containers/PipelineContainer.tsx [hunk 2]
```diff
@@ -12,10 +13,15 @@ export default class PipelineContainer extends ChildContainer<
     super();
     this.state = {
       currentPartner: 'ALL',
+      currentTeam: 'Boston', // 2/ can I use cuc here to do cuc.user.team, or how do I get the current user's current team?
```
Comment (KNWR) [inline client/containers/PipelineContainer.tsx]: 2/ can I use cuc here to do `cuc.user.team`? (in which case I'd be importing CurrentUserContainer at the top), or how do I get the current user's current team?

File: client/containers/PipelineContainer.tsx (hunk 1 above)
Comment (VinaiRachakonda) [reply client/containers/PipelineContainer.tsx]: Team enum

File: client/containers/PipelineContainer.tsx (hunk 2 above)
Comment (VinaiRachakonda) [reply client/containers/PipelineContainer.tsx]: Use cuc here. You should import as necessary

Comment (KNWR) [reply client/containers/PipelineContainer.tsx]: Is this how I'd access `cuc.user` here? Not sure if correct – I see async's used to get userId's in the other container tsx files, nothing like as below

```
import { CurrentUserContainer } from '../../containers/CurrentUserContainer';
```
at the top, and then for the highlighted line here:
```
currentTeam =  <Subscribe to={[CurrentUserContainer]}>
                            {(cuc: CurrentUserContainer) => (cuc.user.team)}
                          </Subscribe>
```

Comment (casey-chow) [reply client/containers/PipelineContainer.tsx]: `cuc.user` via a subscribe component is generally correct. You'll need to use `partnerTeam` though: https://github.com/dormroomfund/magic-wand/blob/master/client/schemas/user.ts#L39

Comment (KNWR) [reply client/containers/PipelineContainer.tsx]: Not sure if makes sense here (not returning html/jsx, just tried to get partnerTeam of current user)- I tried and got a memory leak  issue re: Can't perform a React state update on an unmounted component.

Didn't see cuc called in any of the other containers besides the currentUserContainer itself, so taking a diff approach

Comment (casey-chow) [reply client/containers/PipelineContainer.tsx]: Oh we're in a container, my bad. Every container has access to every other container through the `ctx` property.

File: client/components/Pipeline/Column.tsx [hunk 3]
```diff
@@ -53,12 +54,21 @@ export default class Column extends React.Component<ColumnProps, ColumnState> {
...
-  renderCard = (company, pipe, index, status) => {
-    const shouldDisplay = company.pointPartnersNames.has(
+  renderCard = (company, pipe, index, status, userPartnerTeam) => {
+    const selectedPartnerHasCompany = company.pointPartnersNames.has(
       pipe.state.currentPartner
     );
-    const shouldDisplay2 = shouldDisplay || pipe.state.currentPartner === 'ALL';
-    if (shouldDisplay2) {
+    // TODO: Should not filter this way. Instead, filter the companies shown in
+    // the db query on line 76 in Kanban.tsx
```
Comment (VinaiRachakonda) [inline client/components/Pipeline/Column.tsx:62]: Can you add a general comment of what the filtering process is right now? Can you also add a ticket for this TODO?

**Review by VinaiRachakonda — APPROVED** [review]
LGTM

File: client/components/Pipeline/Column.tsx (hunk 3 above)
Comment (KNWR) [reply client/components/Pipeline/Column.tsx:62]: Done

**Pull request merged**